testing/local-binder-local-hub/binderhub_config.py

```diff
@@ -1,14 +1,18 @@
-######################################################################
-## A development config to test BinderHub locally.
-#
-# If you are running BidnerHub manually (not via JupyterHub) run
-# `python -m binderhub -f binderhub_config.py`
-#
-# Override the external access URL for JupyterHub by setting the
-# environment variable JUPYTERHUB_EXTERNAL_URL
-
-# Host IP is needed in a few places
+"""
+A development config to test BinderHub locally.
+
+If you are running BidnerHub manually (not via JupyterHub) run
```

Comment:
```suggestion
If you are running BinderHub manually (not via JupyterHub) run
```
Might as well fix my typo 😄